src/asymmetric_cipher.c

```diff
@@ -17,13 +19,24 @@ DISPATCH_RSAENC_FN(set_ctx_params);
 DISPATCH_RSAENC_FN(gettable_ctx_params);
 DISPATCH_RSAENC_FN(settable_ctx_params);
 
+struct tls_padding {
+    bool mode;
```

Comment:
I find this member confusing as the name suggest that this could be used to switch different modes of the tls_padding, but it is technically just a knob to enable the padding altogether so I would propose a name like `enabled` or something similar, which would make it more self-explanatory.

Reply:
Done